content/document.js

```diff
@@ -501,20 +579,32 @@ function findChildren(url, recursive = false) {
   const root = getRoot(locale);
   const folder = urlToFolderPath(url);
 
-  const api = new fdir()
-    .withFullPaths()
-    .withErrors()
-    .filter((filePath) => {
-      return (
-        filePath.endsWith(HTML_FILENAME) || filePath.endsWith(MARKDOWN_FILENAME)
-      );
-    })
-    .withMaxDepth(recursive ? Infinity : 1)
-    .crawl(path.join(root, folder));
-  const childPaths = [...api.sync()];
-  return childPaths
-    .map((childFilePath) => path.relative(root, path.dirname(childFilePath)))
-    .map((folder) => read(folder));
+  if (process.env.NODE_ENV !== "production") {
+    const api = new fdir()
```

Comment:
Would probably be good to have a comment explaining why we handle production differently.